DOCSP-30634 Mapping Rule Options Drawer (#21)
mongodb/docs-relational-migrator: source/mapping-rules/mapping-rule-options/new-documents.txt

=============
New Documents
=============

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

The :guilabel:`New Documents` mapping rule is the most basic mapping rule option

> which creates a document for each row of data in your relational database.

Reviewer: (simple sentences)
Consider something like:
"New Documents is the most basic mapping rule option. It creates a document for each row of data in your relational database.

Author: Yes, good suggestion. Updated.

About this Task
---------------

Any database table can use a :guilabel:`New Documents` mapping rule. 

Behavior
--------

Use this mapping rule when you want data from a table to be 
stored in a dedicated collection. 

You can have multiple :guilabel:`New Documents` mapping rules from

> different tables into the same collection. Relational Migrator will treat

Reviewer: (present tense)
```suggestion
different tables into the same collection. Relational Migrator treats
```

Author: Updated.

> these tables as isolated data and the destination documents will not be 

Reviewer: ```suggestion
these tables as isolated data and the destination documents are not 
```

Author: Updated.

merged together in any way. 

Example
-------

This section demonstrates the relational input and
the MongoDB output of the :guilabel:`New Documents` mapping rule.

New Documents
~~~~~~~~~~~~~

In this example a :guilabel:`New Documents` mapping rule is applied to a single row 
from the ``Customer`` table.

Click the tabs below to see the input and output for this example:

> 

Reviewer: (praise) 
I like this use of tabs a lot!
